.github/workflows/ci.yml

```diff
@@ -5,6 +5,58 @@ on:
       - main
   pull_request:
 jobs:
+  black:
+    runs-on: ubuntu-latest
+    container: python:3.8-slim-bullseye
+    steps:
+      - name: Checkout Code Repository
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -q black==22.6.0
+      - name: black
+        run: |
+          black . --check
+  isort:
+    runs-on: ubuntu-latest
+    container: python:3.8-slim-bullseye
+    steps:
+      - name: Checkout Code Repository
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -q isort==5.10.1
+      - name: isort
+        run: |
+          isort . --check-only --profile black
+  flake8:
+    runs-on: ubuntu-latest
+    container: python:3.8-slim-bullseye
+    steps:
+      - name: Checkout Code Repository
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -q flake8==4.0.1 flake8-assertive==2.1.0 flake8-blind-except==0.2.1 flake8-comprehensions==3.10.0
+      - name: flake8
+        run: |
+          flake8 .
+  djhtml:
+    runs-on: ubuntu-latest
+    container: python:3.8-slim-bullseye
+    steps:
+      - name: Checkout Code Repository
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -q djhtml==1.5.1
+      - name: djhtml
+        run: |
+          djhtml wagtailio/**/*.html --check
```

Comment:
this all can be replaced with the pre-commit action

Reply:
https://github.com/torchbox/wagtailmedia/blob/main/.github/workflows/test.yml#L13-L25

Reply:
thanks, updated in 36477dae8fe3b1fcadcc0a1643a21bfb4aeeeea6